.coveragerc

```diff
@@ -44,6 +44,8 @@ omit =
     homeassistant/components/asterisk_mbox/*
     homeassistant/components/asuswrt/device_tracker.py
     homeassistant/components/august/*
+    homeassistant/components/axis/*
```

Comment:
Why did you add this line? Merge issue?

Reply:
Thanks for the review @awarecan! Yes indeed, I saw a couple of merge conflicts that I had to fix and I think I did something wrong there. Should be fixed in 708f9aa90836d53ae9f85f6ad17884a076dfcd37.